backend/src/api/routes/auth/custom-oauth.routes.ts

```
@@ -253,7 +250,11 @@ router.get('/:key', async (req: Request, res: Response, next: NextFunction) => {
       { algorithm: 'HS256', expiresIn: '1h' }
     );
 
-    const authUrl = await customOAuthProvider.generateOAuthUrl(key, state);
+    const authUrl = await customOAuthProvider.generateOAuthUrl(
+      key,
+      state,
+      additionalParams
+    );
```

ESLint

[error] Replace `⏎······key,⏎······state,⏎······additionalParams⏎····` with `key,·state,·additionalParams`

(prettier/prettier)